release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+# Overview of jBallerina 1.2.0
+
+JBallerina 1.2.0 is the first major Q1 release of 2020 and it includes a new set of features and significant improvements to the compiler, runtime, standard library and developer tooling. It is based on Stable Language Spec version 2020R1.
+
+# Highlights
+
+- Redesigned XML, Stream and Query support
+- Based on a stable language specification: 2020R1
+- Improved Compilation time.
+- Includes a number of important bug fixes
+- Added cookies and trailer support in HTTP.
+- Revamped Cache module.
+- Redesigned oneof field and map field support in gRPC and Added client retry support for gRPC blocking unary calls.
+- Added custom serializer/deserializer and Avro support in the Kafka connector. 
+- Deprecated previous JDBC connector and introduced new Database connectors which are now in a preview state.
```

Comment:
Can deprecating something be considered a highlight? That too, to be replaced by something in preview state? I think mentioning the introduction of new DB connectors is sufficient for this section

Reply:
@wggihan @hasithaa

Reply:
+1. We can mention the deprecating blow in the stdlib section.

Reply:
@praneesha this needs to be fixed

Reply:
Changed to "introduced new Database connectors"